openedx/core/djangoapps/user_api/views.py

```diff
@@ -249,6 +251,23 @@ def get(self, request):
         form_desc = FormDescription("post", reverse("user_api_registration"))
         self._apply_third_party_auth_overrides(request, form_desc)
 
+        if configuration_helpers.get_value(
+            'ALLOW_REGISTRATION_FORM_FIELD_OVERRIDE',
+            settings.FEATURES.get('ALLOW_REGISTRATION_FORM_FIELD_OVERRIDE', False)
+        ):
+
+            additional_fields = configuration_helpers.get_value(
+                'REGISTRATION_FIELD_DEFAULTS',
+                settings.FEATURES.get('REGISTRATION_FIELD_DEFAULTS', [])
```

Comment:
Same suggestion [as above](https://github.com/open-craft/edx-platform/pull/120/files#diff-b6f6ec5652c3754ec1c2a13aeb469553R36) to avoid adding new `settings.FEATURES` and just use site configuration instead.